path/to/settings.html

```
@@ -0,0 +1,23 @@
+/**
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)